Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -52,27 +58,35 @@ local YIELD_STATE :table = {
 -- ===========================================================================
 --  GLOBALS
 -- ===========================================================================
+-- ==== CQUI CUSTOMIZATION BEGIN ====================================================================================== --
+-- CQUI uses CQUI_BubbleInstance instead of AmenityInstance
 g_kAmenitiesIM = InstanceManager:new( "CQUI_BubbleInstance", "Top", Controls.AmenityStack );
+-- ==== CQUI CUSTOMIZATION END ======================================================================================== --
 
 -- ===========================================================================
 --  MEMBERS
 -- ===========================================================================
 
-local m_kBuildingsIM = InstanceManager:new( "BuildingInstance", "Top");
-local m_kDistrictsIM = InstanceManager:new( "DistrictInstance", "Top", Controls.BuildingAndDistrictsStack );
-local m_kHousingIM = InstanceManager:new( "CQUI_BubbleInstance", "Top", Controls.HousingStack );
-local m_kOtherReligionsIM = InstanceManager:new( "OtherReligionInstance", "Top", Controls.OtherReligions );
-local m_kProductionIM = InstanceManager:new( "ProductionInstance", "Top", Controls.ProductionQueueStack );
-local m_kReligionsBeliefsIM = InstanceManager:new( "ReligionBeliefsInstance", "Top", Controls.ReligionBeliefsStack );
-local m_kTradingPostsIM = InstanceManager:new( "TradingPostInstance", "Top", Controls.TradingPostsStack );
-local m_kWondersIM = InstanceManager:new( "WonderInstance", "Top", Controls.WondersStack );
-local m_kKeyStackIM = InstanceManager:new( "KeyEntry", "KeyColorImage", Controls.KeyStack );
+local m_kBuildingsIM        :table = InstanceManager:new( "BuildingInstance", "Top");
```

Comment:
unmodified firaxis version of this file has the " :table " for all of these things, so it was added here to match